src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -127,8 +133,8 @@ export class FeederPodcastModel extends BaseModel {
     }
   }
 
-  set(field: string, value: any) {
-    super.set(field, value);
+  set(field: string, value: any, forceOriginal = false) {
```

Comment:
This function got removed from underneath you. Probably should do a merge from master and make sure it's goes away in this branch.